Bottom: 6cc1347bf8fc5d99878e7f11e1201a68187727ab
Top:    96e5153f42640ee68c407603c3b2e37d5f3a9880
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-07 19:17:40 +0000

Avoid clearing pins / vias from polygons... it is already done!





---

diff --git a/src/draw.c b/src/draw.c
index 8143cea..1cacc5c 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1262,6 +1262,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1284,6 +1285,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
